ee/clickhouse/queries/event_query.py

```diff
@@ -82,6 +82,18 @@ def _get_disintct_id_query(self) -> str:
         else:
             return ""
 
+    def get_aggregation_target_field(self) -> str:
```

Comment:
I wonder if this should be a mixin rather than there for every event query. :thinking: 

WDYT about extracting it under `ee/clickhouse/queries/mixins/group_event_query_aggregation.py` or somesuch?

Reply:
Hmm, it's just an event query generation helper, right? And almost every event query needs some aggregation_target. This just moves what you had in the RetentionEventQuery to the base class, so Funnels, Retention (and soon Paths), and Trends (post a future refactor if it happens, to unify things), should be making use of this?

I'm a bit wary of introducing mixins here that then enable new functions to appear for EventQuery classes: If it's specific to a single query, define it in the specificQueryEvent class. If it's common, let it be here. Thoughts?

Reply:
Trends does not use `aggregation_group_type_index` though (instead going through math), which is why I'm wary of introducing something that wouldn't work for it.

Alternatively we could export a function somewhere for this.

Reply:
> which is why I'm wary of introducing something that wouldn't work for it.

It's an available helper, and wouldn't be used unless you call it somewhere in trendsEventQuery?

Exporting as a function is reasonable: I'm just annoyed by all the state that needs to be passed here everytime we want to use it (all of which are indeed part of `EventQuery`). Don't get the benefit this gives us? If you end up using this function in trends as well, it wouldn't work. It's just the place it's located that's different now.